website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -48,39 +61,33 @@ To start the agent and connect it to an HCP Terraform agent pool:
 1. Set the `TFC_AGENT_TOKEN` environment variable.
 1. (Optional) Set the `TFC_AGENT_NAME` environment variable. This name is for your reference only. The agent ID appears in logs and API requests.
 
-```
-export TFC_AGENT_TOKEN=your-token
+```shell
+export TFC_AGENT_TOKEN=${your-token}
 export TFC_AGENT_NAME=your-agent-name
 ./tfc-agent
 ```
-Once complete, your agent and its status appear on the **Agents** page in the HCP Terraform UI. Workspaces can now use this agent pool for runs. Refer to [Configure Workspaces to Use the Agent](/terraform/cloud-docs/agents/agent-pools#configure-workspaces-to-use-the-agent) for details.
-
-### Optional Configuration: Run an Agent Using Docker
-
-Alternatively, you can use our official agent Docker container to run the agent.
-
-```
-docker pull hashicorp/tfc-agent:latest
-docker run -e TFC_AGENT_TOKEN=your-token -e TFC_AGENT_NAME=your-agent-name hashicorp/tfc-agent
-```
 
-This Docker image executes the `tfc-agent` process as the non-root `tfc-agent` user. For some workflows, such as workflows requiring the ability to install software using `apt-get` during `local-exec` scripts, you may need to build a customized version of the agent Docker image for your internal use.
-
-```Dockerfile
-FROM hashicorp/tfc-agent:latest
+Once complete, your agent and its status appear on the **Agents** page in the HCP Terraform UI. Workspaces can now use this agent pool for runs. Refer to [Configure Workspaces to Use the Agent](/terraform/cloud-docs/agents/agent-pools#configure-workspaces-to-use-the-agent) for details.
+~> **Important:** At this time, starting an agent with this method is not supported by AMD. Using the official agent Docker image instead is recommended, with the `--platform` tag set.
```

Comment:
i don't think we need this here yet since i don't think the arm stuff is live yet so idk if the platform tag is something folks can use yet. we also have the current supported os already documented on the requirements page https://developer.hashicorp.com/terraform/cloud-docs/agents/requirements#supported-operating-systems
```suggestion
```